sdk/src/wallet/account/mod.rs

```diff
@@ -567,3 +567,27 @@ fn serialize() {
 
     serde_json::from_str::<AccountDetails>(&serde_json::to_string(&account).unwrap()).unwrap();
 }
+
+#[cfg(test)]
+impl AccountDetails {
+    /// Returns the following account details:
+    /// index: 0, coin_type: 4218, alias: "Alice", all fields containing arrays/maps are empty
+    pub(crate) fn mock() -> Self {
+        Self {
+            index: 0,
+            coin_type: 4218,
+            alias: "Alice".to_string(),
+            public_addresses: Vec::new(),
```

Comment:
During account creation the first public address is generated and in other parts of the code we expect that there exists at least one, so I think to make it more future prove it would be good to also add one here

Reply:
good suggestion :+1: